SPEC.md

```diff
@@ -65,21 +65,21 @@ Examples: `minitest`, `mysql2`, `battery-horse`.
 
 The version number of the library (String, required).
 
-- It should follow the [Semantic Versioning](http://semver.org/) format.
+- It should follow the [Semantic Versioning 2.0](http://semver.org/spec/v2.0.0.html) format.
```

Comment:
Really? I would have argued the opposite due to the nature of the semver spec following semantic versioning.

Or is the plan for this project to always support only the latest semver spec?